### PR TITLE
Add ability to configure 'target' behavior of rich text content.

### PR DESCRIPTION
**Pull request by tmeyer2115**
This PR adds a 'target' parameter to the SDK's formatRichText function. This
parameter, when specified, sets the 'target' attribute of all the formatted RTF
links. It is optional.

J=SPR-2134
TEST=auto, manual

Added unit tests for the updated formatRichText function. Manually verified:
- Passing a 'target' of '_blank' to formatRichText added the correct 'target'
  attribute to all of the link anchor tags.
- When I did not provide a 'target' to formatRichText, the link anchor tags had
  no 'target' attribute.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1860,13 +1860,16 @@ The Answers SDK exposes a `formatRichText` function which translates CommonMark
 ensure that a Rich Text Formatted value is shown properly on the page. To use this function, call it like so:
 
 ```js
-ANSWERS.formatRichText(rtfFieldValue, eventOptionsFieldName)
+ANSWERS.formatRichText(rtfFieldValue, target, eventOptionsFieldName)
```
Comment (oshi97) [inline README.md]: would we want to generalize this to an attributes param object, that lets you stick arbitrary attributes onto the html, or is that overengineered?

Comment (tmeyer2115) [reply README.md]: We currently don't have requirements for that. The HHs are really only interested in changing the 'target' attribute of a link.

**Review by oshi97 — APPROVED** [review]

**Review by oshi97 — APPROVED** [review]
lgtm! I guess whenever we want to use a specific plugin we have to do this add and disable pattern?

**Comment by tmeyer2115** [comment]
> lgtm! I guess whenever we want to use a specific plugin we have to do this add and disable pattern?

Yep! If you're okay with the plugin you added being used during subsequent RichTextFormatter#format calls, no need to disable!

**Pull request merged**